setup.cfg

```diff
@@ -0,0 +1,38 @@
+[metadata]
+name = openfe
+version = 0.0.1.dev0
+description = 
+long_description = file: README.md
+long_description_type = text/markdown
+#author =
```

Comment:
I'd also advocate for "OpenFE developers" + "openfe@omsf.io" here.

Reply:
I'll need to do some tinkering to make openfe@omsf.io redirect properly to all current devs